src/sql/workbench/contrib/notebook/browser/notebookEditor.ts

```diff
@@ -283,58 +283,58 @@ export class NotebookEditor extends BaseEditor implements IFindNotebookControlle
 			} else {
 				this._finder.getDomNode().style.visibility = 'hidden';
 				this._findDecorations.clearDecorations();
+				return;
```

Comment:
Why the returns here?

Reply:
no need to further check when we're already handling the reveal event here, below events needs to be checked only when isRevealed is true.